src/core/formatters.ts

```diff
@@ -131,6 +131,54 @@ export module Formatters {
   }
 
   /**
+   * Creates a formatter for values that displays abbreviated values
+   * and uses standard short scale suffixes (thousands, millions, billions, trillions,
+   * quadrillions).
+   *
+   * Numbers with a magnitude outside of (10^-precision, 10^(15+precision)) are shown using
+   * scientific notation to avoid creating extremely long decimal strings.  The inputs to the
+   * formatter are primarily expected to be ranges of common currency values.
+   *
+   * @param {number} [precision] the number of decimal places to show (default 3)
```

Comment:
I tried out the fiddle, and it looks like it always returns exactly `precision` decimal places.